assets/src/scss/pages/post/_article-content.scss

```diff
@@ -284,3 +284,23 @@
     text-align: center;
   }
 }
+
+.post-content .article-list-item {
+  .top-page-tags {
+    margin-bottom: $sp-1;
+  }
+
+  .article-list-item-headline {
+    padding-bottom: 0;
+    margin-bottom: $sp-1;
+  }
+
+  .article-list-item-content {
```

Comment:
There is a similar behavior [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/PLANET-6723_allow_articles_block_in_posts/assets/src/styles/blocks/Articles.scss#L125-L134) but only applied to `mobile`. In our case we need to apply new styles only when the Article is included into a post.